docs/share_models.rst

```diff
@@ -38,7 +38,18 @@ Creative Work
         class CreativeWork:
             awards = Map(Delegate(ThroughAwards), ctx.<awards>)
   - venues
-   - A list of venues associated with the work, passed to the ``Venue`` class via the ``ThroughVenues`` class (syntax follows the ``awards`` example above).
+   - A list of venues associated with the work, passed to the ``Venue`` class via the ``ThroughVenues`` class::
+   
+        class Venue:
+            description = ctx.<venue_description>
```

Comment:
Since this looks to be a runnable code example, the angle-brackets should be removed to make this valid Python. I'll do this when I merge.